src/stories/button.stories.js

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import Button from "../components/button"
+import { action } from "@storybook/addon-actions"
+
+export default {
+  title: "Button",
+  component: Button,
+  args: {
+    buttonName: "Default",
```

Comment:
this is related to the comment below.. if we decide to add the title by convention, we could generalize the name here a bit, like`componentTitle`